File: Looping/Loop inside loop/LoopInLoop2.java
public class LoopInLoop2 {
	public static void main(String[] args) throws Exception {
		System.out.print("\n");
		for (int vert = 1; vert <= 4; vert++) {
			// Nested Loops gave me anxiety...
			for (int horz = 1; horz <= 8; horz++) {
				System.out.print(horz + " ");
			}
			System.out.print("\n");
		}
	}
}

File: Looping/Loop inside loop/Other Lang/C/LoopInLoop2.c
#include <stdio.h>

void main() {
	for (int vert = 1; vert <= 4; vert++) {
		//Nested Loops Gave Me Anxiety...
		for (int horz = 1; horz <= 8; horz++) {
			printf("%d ", horz);
		}
		printf("\n");
	}
}
